### PR TITLE
Added message history and message entity

### PR DESCRIPTION
**Pull request by bw55555**
Added message entity to consolidate text and options
Added message history entity to store last message
Added methods to use message entity instead of text/options

### CODE REVIEW

**Review by Yasutakahi — APPROVED** [review]
small fix but very reasonable and nice that you added doc strings.

File: src/main/java/io/Message.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+    private final String text;
+    private final List<String> options;
+
+    public Message(String text, List<String> options) {
+        this.text = text;
+        this.options = options;
+    }
+
```
Comment (mahirhassan-uoft) [inline src/main/java/io/Message.java:20]: add doctests here

**Review by mahirhassan-uoft — APPROVED** [review]
well done! :D

**Review by marjaleenakiik — APPROVED** [review]
Added docstrings are descriptive and help with understanding the code. The Message class makes sense, helps with returning to a previous ask for a user's input.

**Pull request merged**